src/resolvers/path.cr

```diff
@@ -56,15 +48,15 @@ module Shards
       {% end %}
     end
 
-    def available_releases : Array(String)
-      [spec.version] of String
+    def available_releases : Array(Version)
+      [spec(nil).version]
```

Comment:
I guess the default value in `def spec(version = nil)` can go away.

Reply:
Maybe. Actually I want to refactor a little bit more the methods that read specs from files. There is some duplication now.